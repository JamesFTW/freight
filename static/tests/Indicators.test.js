import React from 'react';
import renderer from 'react-test-renderer';
import Indicators from '../components/Indicators.jsx';


test('Indicators Snapshot', () => {
  const component = renderer.create(
    <Indicators />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
