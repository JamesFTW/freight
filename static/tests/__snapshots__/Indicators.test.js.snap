// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indicators Snapshot 1`] = `<div />`;
